organisaatio-service/src/main/java/fi/vm/sade/organisaatio/dao/impl/OrganisaatioDAOImpl.java

```diff
@@ -336,12 +339,15 @@ public boolean isYtunnusAvailable(String ytunnus) {
     }
 
     @Override
-    public List<Organisaatio> findModifiedSince(Date lastModifiedSince) {
+    public List<Organisaatio> findModifiedSince(Boolean piilotettu, Date lastModifiedSince) {
         LOG.debug("findModifiedSince({})", lastModifiedSince);
 
         QOrganisaatio qOrganisaatio = QOrganisaatio.organisaatio;
 
         BooleanExpression whereExpression = qOrganisaatio.paivitysPvm.after(lastModifiedSince);
+        if(piilotettu != null){
+            whereExpression.and(qOrganisaatio.piilotettu.eq(piilotettu));
```

Comment:
Tässä pitääkin olla `whereExpression = whereExpression.and(qOrganisaatio.piilotettu.eq(piilotettu));`